docs/en/Running-Alluxio-On-Kubernetes.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: global
+title: Running Alluxio on Kubernetes
+nickname: Alluxio on Kubernetes
+group: Deploying Alluxio
+priority: 3
+---
+
+Alluxio can be run on Kubernetes. This guide demonstrates how to run Alluxio
+on Kubernetes using the specification that comes in the Alluxio Github repository.
+
+# Basic Tutorial
+
+This tutorial walks through a basic Alluxio setup on Kubernetes.
+
+## Prerequisites
+
+- A Kubernetes cluster (version >= 1.8). Alluxio workers will use `emptyDir` volumes with a 
+restricted size using the `sizeLimit` parameter. This is an alpha feature in Kubernetes 1.8.
+Please ensure the feature is enabled.
+- An Alluxio docker image. Refer to [this page](Running-Alluxio-On-Docker.html) for instructions
+to build an image.
+
+## Clone the Alluxio repo
+
+```bash
+$ git clone https://github.com/Alluxio/alluxio.git
+$ cd integration/kubernetes
+```
+
+The kubernetes specifications required to deploy Alluxio can be found under `integration/kubernetes`.
+
+## Short-circuit operations
```

Comment:
it would be nice if each header in the guide is a verb - this could be "Enable short-circuit operations" or "Set up shared domain socket"